plugins/convert/masked.py

```diff
@@ -43,6 +43,8 @@ def patch_image(self, image, detected_face):
         detected_face.load_aligned(image, size=self.training_size, align_eyes=False)
         new_image = self.get_new_image(image, detected_face, coverage, image_size)
         image_mask = self.get_image_mask(detected_face, image_size)
+        if self.args.draw_transparent:
+            new_image = dfl_full(detected_face.landmarks_as_xy, new_image, channels=4)
```

Comment:
The next logic line in the code is in apply fixes and is a boolean
 ```if self.args.draw_transparent:```
Might as well put line 47 at line 157

Reply:
This is necessary here since alignments aren't sent to apply_fixes().  Moved those lines up instead.